test/Microsoft.Sbom.Parsers.Spdx30SbomParser.Tests/Parser/JsonStrings/SbomFileJsonStrings.cs

```diff
@@ -103,7 +103,7 @@ public static class SbomFileJsonStrings
             ""to"": [
               ""SPDXRef-AnyLicenseInfo-3BA3FA6D3D66FE2BA75992BB0850D080F1223256368A76C77BEF8E0F6AC71896""
             ],
-            ""creationInfo"": ""_:creationinfo"",
+            ""creationInfo"": ""_:creationinfo"",            
```

Comment:
Trim trailing spaces?
```suggestion
            ""creationInfo"": ""_:creationinfo"",
```